week-2-practise/level-2/6-map.js

```diff
@@ -4,7 +4,10 @@
 //    - each item in `percentages` formatted by your function
 
 var percentages = [1, 23, 92, 18];
-
+function persentage(index) {
+  return index + "%";
+}
```

Comment:
Small typo, it should be `percentage` instead of `persentage`. More importantly though it gives the right answer. Every exercise in level 1 and level 2 has produced the right results so far. Very very good.

Reply:
Hahaha I see now